aspnetcore/signalr/scale.md

```diff
@@ -0,0 +1,87 @@
+---
+title: ASP.NET Core SignalR production hosting and scaling
+author: tdykstra
+description: Learn how to avoid performance and scaling problems in apps that use ASP.NET Core SignalR.
+monikerRange: '>= aspnetcore-2.1'
+ms.author: tdykstra
+ms.custom: mvc
+ms.date: 11/28/2018
+uid: signalr/scale
+---
+
+# ASP.NET Core SignalR hosting and scaling
+
+By [Andrew Stanton-Nurse](https://twitter.com/anurse), [Brady Gaster](https://twitter.com/bradygaster), and [Tom Dykstra](https://github.com/tdykstra),
+
+This article explains hosting and scaling considerations for high-traffic apps that use ASP.NET Core SignalR.
+
+## TCP connection resources
+
+The number of concurrent TCP connections that a web server can support is limited. Standard HTTP requests use these limited resources efficiently, opening and closing a connection for every request and response. A SignalR connection holds the TCP connection open for a much longer time. In a high-traffic app that serves many clients, these persistent connections can cause servers to hit their maximum number of connections.
```

Comment:
```suggestion
The number of concurrent TCP connections that a web server can support is limited. Standard HTTP clients use *ephemeral* connections. These connections can be closed when the client goes idle and re-opened later. On the other hand, a SignalR connection is *persistent*. SignalR connections stay open even when the client goes idle. In a high-traffic app that serves many clients, these persistent connections can cause servers to hit their maximum number of connections.
```

Had another thought about rephrasing this. We also want to avoid the nit-pickers who say that HTTP requests do use connections "efficiently" and don't just keep opening and closing connections ;).